### PR TITLE
Align License rp_module_licence format to RetroPie-Setup

### PR DESCRIPTION
**Pull request by Gemba**
This aligns the license header information from `rp_module_licence=` to be more RetroPie-Setup alike by using the same key for commonly used licences.

BTW: Is there a reason you put this repo at MIT licence while the RetroPie-Setup uses GPL3? Why not relicence to GPL3 as it grants more freedom?

### CODE REVIEW

**Comment by s1eve-mcdichae1** [comment]
The repo license came with the fork, as originally penned by ZeroJay.

The one at RetroPie is a lot more...specific. This one is much simpler to understand, at least. I absolutely do not know which one is "better" or if it's even allowed to just "relicense" the work that was originally made by someone else. If the idea is to grant "more freedom," wouldn't it have to be the initial license holder ZJ who does that?

File: scriptmodules/ports/devilutionx.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 rp_module_id="devilutionx"
 rp_module_desc="devilutionx - Diablo Engine"
-rp_module_licence="https://raw.githubusercontent.com/diasurgical/devilutionX/master/LICENSE"
+rp_module_licence="SUL1 https://github.com/diasurgical/devilutionX/blob/master/LICENSE.md"
```
Comment (s1eve-mcdichae1) [inline scriptmodules/ports/devilutionx.sh:15]: Most times they use the raw url: https://raw.githubusercontent.com/diasurgical/devilutionX/master/LICENSE.md

**Comment by Gemba** [comment]
Makes sense. 
Changed four modules to have the license referenced the "RetroPie-way".

**Comment by Exarkuniv** [comment]
submit this on the guy who scalped my work and is putting it off like he did it all

**Pull request closed without merging**